### DISCUSSION TITLE
1 map left in my list with a permanent update button

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#19495] [Q&A] [answered]
[This mapset](https://osu.ppy.sh/beatmapsets/24632#osu/95413) seems to be the only one on my list that still has the button to update it (despite being untouched). I've tried deleting it, restarting the game, and re-downloading it to ensure its genuineness.

![image](https://user-images.githubusercontent.com/43073074/182021775-e7777d94-d7a9-4b55-a191-a28bfd85b8ea.png)

**Comment by frenzibyte** [comment]
The MD5 hash of the beatmap online is shown to be `1e0e3054be8953ab9fc98dd9f2651914`:
<img width="528" alt="CleanShot 2022-08-01 at 11 49 06@2x" src="https://user-images.githubusercontent.com/22781491/182110718-357ca099-6300-4e67-9e17-fb6637d7c507.png">

While the hash of the actual file is `bf9cdc43dfa7ea53a55bab9bfc3e177a`:
```
╰─λ md5 Miani*                                                                                0 (0.005s)
MD5 (Miani - Tu Vivi Nell'aria (Ivan Fillini Remix) (-Lennox-) [Hard].osu) = bf9cdc43dfa7ea53a55bab9bfc3e177a
```

cc @peppy

**Comment by Ephemeralis** [comment, marked as answer]
This should be fixed now after we force-updated the osz file to contain the most recent .osu files.

**Reply by uzervlad** [reply]
The same issue on [this map](https://osu.ppy.sh/beatmapsets/370819)

**Reply by peppy** [reply]
@Ephemeralis can you check on that one too? ^

**Reply by ghost** [reply]
Apologies for the late reply but I have the same issue on [Taihen na Mono no Shoushitsu](https://osu.ppy.sh/beatmapsets/17819)
